README.md

```diff
@@ -21,9 +21,9 @@ To know more about Zeppelin, visit our web site [http://zeppelin.apache.org](htt
 ## Getting Started
 
 ### Install binary package
-Please go to [install](https://zeppelin.apache.org/docs/latest/install/install.html) to install Apache Zeppelin from binary package.
+Please go to [install](http://zeppelin.apache.org/docs/latest/quickstart/install.html) to install Apache Zeppelin from binary package.
```

Comment:
why https -> http?

Reply:
Ah right, pushed a commit to change http->https.